content/en/cosign/timestamps.md

```diff
@@ -0,0 +1,7 @@
+---
+title: "Timestamps"
+category: "Cosign"
+position: 113
+---
+
```

Comment:
Let's note that this is a WIP. Can you add the line:

Note: This document is a work in progress.